LICENSE

```diff
@@ -0,0 +1,10 @@
+Copyright 2013 Google Inc. All Rights Reserved.
```

Comment:
I think here you should use a verbatim copy of the apache2 licesnse, copy it from another app